Readme.md

```diff
@@ -8,19 +8,25 @@
 **[[Run Online]](https://codecentral.devexpress.com/128540794/)**
 <!-- run online end -->
 
-![Delete Confirmation Dialog](delete-confirm-dialog.png)
+This example demonstrates how to implement a delete confirmation dialog with an option to prevent its displays for subsequent grid row deletions.
```

Comment:
```suggestion
This example demonstrates how to implement a delete confirmation dialog with an option to prevent its display for subsequent grid row deletions.
```